0010-endpoint-metadata-extension/README.md

```diff
@@ -292,10 +192,6 @@ The request payload MUST be a UTF-8 encoded JSON object with the JSON Schema def
   "description": "10/EPMDX get metadata request",
   "type": "object",
   "properties": {
-    "id": {
-      "type": "integer",
-      "description": "The message identifier used to match server response to the request"
-    },
     "keys": {
```

Comment:
Maybe we change the request payload to contain only keys array?
For example,
```json
["key1", "key2"]
```
And in case when client requests all keys it will sent a zero-bytes request